src/NumericalAlgorithms/Interpolation/PredictedZeroCrossing.cpp

```diff
@@ -0,0 +1,17 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "NumericalAlgorithms/Interpolation/PredictedZeroCrossing.hpp"
+#include "NumericalAlgorithms/Interpolation/LinearLeastSquares.hpp"
+
+#include <vector>
+namespace intrp {
+
+double predicted_zero_crossing_value(const std::vector<double>& x_values,
+                             const std::vector<double>& y_values) {
```

Comment:
formatting/indentation

Reply:
I suggest that you just run clang-format on everything...that will take care of such formatting issues automatically